.circleci/config.yml

```diff
@@ -47,6 +47,44 @@ commands:
       - run: npm install --legacy-peer-deps
       - run: npm install -g yarn
 
+  install-bundle-dependencies:
+    parameters:
+      directory:
+        type: string
+        default: .
+    steps:
+      - restore_cache:
+          keys:
+            # This cache key should be updated whenever we change the Ruby version
+            - v4-gem-cache-{{ checksum "Gemfile.lock" }}-{{ arch }}-ruby-3.2.0
+      # CircleCI images fail on Ruby 3.3
+      # https://github.com/fastlane/fastlane/issues/21794#issuecomment-2021331335
+      # Also, not using macos/switch-ruby because its broken
+      - run:
+          # We're using Ruby 3.2.0 here to avoid an issue that caused a crash with Fastlane and Ruby 3.2.2.
+          name: Set Ruby 3.2.0
+          command: |
+            eval "$(rbenv init -)"
+            rbenv install -s 3.2.0  # Replace with your required Ruby version
+            rbenv global 3.2.0
+            rbenv rehash
+      - run:
+          name: Verify Ruby version
+          command: ruby -v
+      - run:
+          name: Bundle install
+          working_directory: << parameters.directory >>
+          command: |
+            bundle config set --local clean 'true'
+            bundle config set --local path 'vendor/bundle'
+            bundle install
```

Comment:
should we keep using `revenuecat/install-gem-mac-dependencies` for this? and install ruby in a separate command?

Reply:
Right, I mostly copied the command we use in purchases-ios in a bunch of places. I thought we might want to move this command to the orb at some point, since usually you can just install ruby and install dependencies together... but I guess it does provide more flexibility that way. I will do that change.